### PR TITLE
Removed team member

### PR DESCRIPTION
**Pull request by DavidKohav**
#370

### CODE REVIEW

**Comment by DavidKohav** [comment]
selection_004

File: _config.yml [hunk 1]
```diff
@@ -156,15 +156,6 @@ authors:
       github: shahardo
       avatar: male.jpg
       image: male.jpg
-  - YuriGizra:
```
Comment (amitaibu) [inline _config.yml]: don't remove, just set to `is_member: false`

Comment (amitaibu) [inline _config.yml]: actually, he has no posts.

Comment (amitaibu) [inline _config.yml]: so please also remove his images from the repo

**Comment by DavidKohav** [comment]
Can I merge after Travis?

**Comment by amitaibu** [comment]
:+1:

**Comment by DavidKohav** [comment]
merge manually

**Pull request closed without merging**